### PR TITLE
Update "Troubleshooting" document

### PR DESCRIPTION
**Pull request by ybiquitous**
- Add a new case of "Analysis never starts" about too many files.
- Remove the section about "Required" status because of its inaccuracy.

### CODE REVIEW

**Comment by ybiquitous** [comment]
Even if a branch is set to "Required", I can see Sider starts an analysis... 🤔

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -66,19 +66,20 @@ linter:
...
-- Sider is configured as a "Required" status check on GitHub.
+### A pull request's author does not have a seat
 
-![Never Start Analysis](./assets/never-start-analysis.png)
+A pull request's author must sign up to Sider and be assigned a seat by organization admins.
+Unless the condition is satisfied, an analysis never starts.
+To solve the problem, we recommend you to try the following:
 
-In order to solve this problem, we recommend you to try the following:
+1. Assign a seat to the pull request's author that has signed up to Sider.
```
Comment (yykamei) [inline docs/troubleshooting.md:79]: [nitpick] I prefer "who" in this context.

```suggestion
1. Assign a seat to the pull request's author who has signed up to Sider.
```

**Review by yykamei — APPROVED** [review]
:+1:

File: docs/troubleshooting.md (hunk 1 above)
Comment (ybiquitous) [reply docs/troubleshooting.md]: Thanks!

**Pull request merged**